### ISSUE TITLE
Failed to initialize (Newest Space Engineers Update 10/14/2024)

### ISSUE THREAD

**Issue by NingyMD** [viktor-ferenczi/performance-improvements#69]
My Error Log:

```
2024-10-14 16:48:38.138 - Thread:   1 ->  Info: PerformanceImprovements: Loading
2024-10-14 16:48:38.202 - Thread:   1 ->  Debug: PerformanceImprovements: Scanning for conflicting code changes
2024-10-14 16:48:38.714 - Thread:   1 ->  Critical: PerformanceImprovements: Detected conflicting code changes:
2024-10-14 16:48:38.720 - Thread:   1 ->  Info: PerformanceImprovements: Code change in Sandbox.Game.Entities.MyCubeGrid Sandbox.Game.Entities.MyCubeGrid::MergeGridInternal(Sandbox.Game.Entities.MyCubeGrid gridToMerge, VRageMath.MatrixI& transform, System.Boolean disableBlockGenerators); expected ddf218c3, actual a2674a04
2024-10-14 16:48:38.720 - Thread:   1 ->  Info: PerformanceImprovements: Code change in System.Void Sandbox.Game.Entities.MyCubeGrid::MechanicalConnectionBlockAttachUpdateStatusChanged(Sandbox.Game.Entities.Blocks.MyMechanicalConnectionBlockBase mechConBlock); expected b0648511, actual bda9b059
2024-10-14 16:48:38.720 - Thread:   1 ->  Info: PerformanceImprovements: Code change in virtual System.Void Sandbox.Game.Entities.MyCubeGrid::OnAddedToScene(System.Object source); expected 3a7fe2cd, actual b5021de4
2024-10-14 16:48:38.720 - Thread:   1 ->  Info: PerformanceImprovements: Code change in System.Void Sandbox.Game.Entities.MyCubeGrid::NotifyBlockAdded(Sandbox.Game.Entities.Cube.MySlimBlock block); expected d33dc30a, actual 2c18b95e
2024-10-14 16:48:38.720 - Thread:   1 ->  Info: PerformanceImprovements: Code change in System.Void Sandbox.Game.Entities.MyCubeGrid::NotifyBlockRemoved(Sandbox.Game.Entities.Cube.MySlimBlock block); expected 9705f749, actual d0b27a63
2024-10-14 16:48:38.720 - Thread:   1 ->  Info: PerformanceImprovements: Code change in Sandbox.Game.Entities.MyCubeGrid Sandbox.Game.Entities.MyCubeGrid::MergeGridInternal(Sandbox.Game.Entities.MyCubeGrid gridToMerge, VRageMath.MatrixI& transform, System.Boolean disableBlockGenerators); expected ddf218c3, actual a2674a04
2024-10-14 16:48:38.720 - Thread:   1 ->  Info: PerformanceImprovements: Code change in Sandbox.Game.Entities.MyCubeGrid Sandbox.Game.Entities.MyCubeGrid::MergeGridInternal(Sandbox.Game.Entities.MyCubeGrid gridToMerge, VRageMath.MatrixI& transform, System.Boolean disableBlockGenerators); expected ddf218c3, actual a2674a04
2024-10-14 16:48:38.726 - Thread:   1 ->  Critical: PerformanceImprovements: Failed to scan for conflicting code changes
[Exception] Detected conflicting code changes
Method: Boolean HarmonyPatchAll(Shared.Logging.IPluginLogger, HarmonyLib.Harmony, Boolean)
Traceback:
   at Shared.Patches.PatchHelpers.HarmonyPatchAll(IPluginLogger log, Harmony harmony, Boolean handleExceptions)
2024-10-14 16:48:38.733 - Thread:   1 ->  [PluginLoader] [Error] Failed to initialize viktor-ferenczi/performance-improvements|Performance Improvements because of an error: System.Exception: Detected conflicting code changes
   at Shared.Patches.PatchHelpers.HarmonyPatchAll(IPluginLogger log, Harmony harmony, Boolean handleExceptions)
   at ClientPlugin.Plugin.Init(Object gameInstance)
   at avaness.PluginLoader.PluginInstance.Init(Object gameInstance)
2024-10-14 16:48:43.322 - Thread:  14 ->  MySandboxGame.PerformPreload() - END
```
   
   
![image](https://github.com/user-attachments/assets/a35b0b09-629c-4443-8997-406694c0dd3b)

**Comment by NingyMD** [comment]
There is also a Multigrid Projector Error. I will submit the logs for the error in a issue there.

**Comment by viktor-ferenczi** [comment]
Version 1.11.11 released, that's compatible with game version 1.205.023
